docs/antora-site-structure.md

```diff
@@ -0,0 +1,127 @@
+# Antora Site Structure for Docs
+[link-asciidoc]: https://docs.asciidoctor.org/asciidoc/latest/
+[link-antora]: https://antora.org
+[link-playbook]: https://docs.antora.org/antora/latest/playbook/
+[link-ui]: https://docs.antora.org/antora-ui-default/
+[link-resource-id]: https://docs.antora.org/antora/latest/page/resource-id-coordinates/
+[link-standard-directories]: https://docs.antora.org/antora/latest/standard-directories/
+[link-antora-yml]: https://docs.antora.org/antora/latest/component-version-descriptor/
+[link-site-yml]: https://docs.antora.org/antora/latest/playbook/#whats-an-antora-playbook
+[custom-attrib-link]: https://docs.antora.org/antora/latest/page/attributes/#custom-attributes
+
+**Table of Contents**
+1. [Why Antora](#why-antora)
+2. [Scope of doc Repositories](#scope-of-doc-repositories)
+3. [Scope of Antora Definitions](#scope-of-antora-definitions)
+4. [Scope of Content Accessability](#scope-of-content-accessability)
+5. [Structure of Directories](#structure-of-directories)
+6. [Scope of Antora Definitions](#scope-of-antora-definitions)
+
+## Why Antora
+
+The main reasons for using [Antora][link-antora] is the following:
+
+1. It uses and extends the [asciidoc][link-asciidoc] text writing format
+
+2. It extends asciidoc with [multi-repo][link-playbook] capabilities. Doing so, the writer does not need to take care about repos anymore, as this is virtualized by Antora and content can be accessed with a standardized way
+
+3. Antora seperates writing the documentation and the [UI-Template][link-ui] to define how the content is presented
+
+## Scope of doc Repositories
+
+The ownCloud documentation is made of a master repo named `docs` which includes additional product repos like the clients or others (content sources, defined in `site.yml`). While each repo can be build individually for testing the outcome, only the build of docs creates a documentation which is pushed to the web.
+
+```
+layer-1     layer-2
+master      included repos
+
+            docs-client-desktop
+docs    --> docs-client-ios-app 
+            docs-client-android
+            ...
+
+```
+
+## Scope of Content Accessability
+
+Because Antora is capable defining additional ressources, you can access content from these resources where they are defined. To do so, you follow the [Resource ID Coordinates][link-resource-id] scheme.
```

Comment:
```suggestion
Because Antora is capable of defining additional resources, you can access content from these resources. To do so, follow the [Resource ID Coordinates][link-resource-id] scheme.
```